### ISSUE TITLE
There is no possible way to write multiline byte string.

### ISSUE THREAD

**Issue by proofit404** [wemake-services/wemake-python-styleguide#565]
# Bug report

Consider we have this code snippet:

```python
@pytest.fixture
def jpeg_image():
   """The smallest possible JPEG image.jpg file."""
   return (
       b'\xff\xd8\xff\xdb\x00C\x00\x03\x02\x02\x02\x02\x02\x03\x02\x02\x02'
       b'\x03\x03\x03\x03\x04\x06\x04\x04\x04\x04\x04\x08\x06\x06\x05\x06'
       b'\t\x08\n\n\t\x08\t\t\n\x0c\x0f\x0c\n\x0b\x0e\x0b\t\t\r\x11\r\x0e'
       b'\x0f\x10\x10\x11\x10\n\x0c\x12\x13\x12\x10\x13\x0f\x10\x10\x10\xff'
       b'\xc9\x00\x0b\x08\x00\x01\x00\x01\x01\x01\x11\x00\xff\xcc\x00\x06'
       b'\x00\x10\x10\x05\xff\xda\x00\x08\x01\x01\x00\x00?\x00\xd2\xcf '
       b'\xff\xd9'
   )
```

## What's wrong

There is no possible way to format code example above to make all linter rules pass.

**Comment by sobolevn** [comment]
We now ignore `W504` in our configuration by default. And advise users to do the same.

That's how your code should look like:
```python
byte_array = (
    b'\xff\xd8\xff\xdb\x00C\x00\x03\x02\x02\x02\x02\x02\x03\x02\x02\x02' +
    b'\x03\x03\x03\x03\x04\x06\x04\x04\x04\x04\x04\x08\x06\x06\x05\x06' +
   ...
```
